src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -1,12 +1,136 @@
 import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
 
+import { useState, useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+
+import generalInfo from '~/assets/img/tutor-home-page/become-tutor/general-info.svg'
+import AppTextField from '~/components/app-text-field/AppTextField'
+import AsyncAutocomplete from '~/components/async-autocomlete/AsyncAutocomplete'
 import { styles } from '~/containers/tutor-home-page/general-info-step/GeneralInfoStep.styles'
+import { LocationService } from '~/services/location-service'
+import { userService } from '~/services/user-service'
+import getEmptyArrayData from '~/utils/get-empty-array-data'
 
 const GeneralInfoStep = ({ btnsBox }) => {
+  const { t } = useTranslation()
+  const [firstName, setFirstName] = useState('')
+  const [lastName, setLastName] = useState('')
+  const [country, setCountry] = useState('')
+  const [city, setCity] = useState('')
+  const [textField, setTextField] = useState('')
+  const [charCount, setCharCount] = useState(0)
```

Comment:
Maybe it is better to use only `textField` state without additional `charCount`, because `charCount`===`textField.length`

Reply:
I've change TextField into AppTextArea